external/dir.targets

```diff
@@ -0,0 +1,20 @@
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
Now that this is a test-only thing, should these files be moved under `/tests/` to make it clear what this is used for?

Reply:
I was aiming to keep consistency with Corefx. I'll stick it under tests to not pollute the root folder so much.